areas/storage/src/AzureMcp.Storage/Services/StorageService.cs

```diff
@@ -314,6 +315,13 @@ private async Task<DataLakeServiceClient> CreateDataLakeServiceClient(string acc
         return new DataLakeServiceClient(new Uri(uri), await GetCredential(tenant), options);
     }
 
+    private async Task<QueueServiceClient> CreateQueueServiceClient(string accountName, string? tenant = null, RetryPolicyOptions? retryPolicy = null)
+    {
+        var uri = $"https://{accountName}.queue.core.windows.net";
```

Comment:
Same comment on 'other' cloud urls

Reply:
Appears to be captured by https://github.com/Azure/azure-mcp/issues/145